tutorials/preprocessing/plot_15_handling_bad_channels.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-bad-channels:
+
+Interpolating bad channels
+==========================
+
+This tutorial covers manual marking of bad channels and reconstructing bad
+channels based on good signals at other sensors.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+As usual we'll start by importing the modules we need, and loading some example
+data:
+"""
+
+import os
+import mne
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, verbose=False)
+
+###############################################################################
+# Marking bad channels
+# ^^^^^^^^^^^^^^^^^^^^
+#
+# Sometimes individual channels malfunction and provide data that is too noisy
+# to be usable. MNE-Python makes it easy to remove bad channels from the
+# analysis stream without actually deleting the data in those channels, by
+# keeping track of the bad channel indices in a list and looking at that list
+# when doing analysis or plotting tasks. The list of bad channels is stored in
+# the `'bads'` field of the :class:`~mne.Info` object that is attached to
+# :class:`~mne.io.Raw`, :class:`~mne.Epochs`, and :class:`~mne.Evoked` objects.
+
+print(raw.info['bads'])
+
+###############################################################################
+# Here you can see that the :file:`.fif` file we loaded from disk must have
+# been keeping track of channels marked as "bad" — which is good news, because
+# it means any changes we make to the list of bad channels will be preserved
+# if we save our data at intermediate stages and re-load it later. In the case
+# of the example data, you can see why those channels were marked bad by
+# using the standard :meth:`~mne.io.Raw.plot` method; to make it easier to see
+# we'll use the :func:`~mne.pick_channels_regexp` function to narrow down which
+# channels we see:
+
+# sphinx_gallery_thumbnail_number = 2
+for pattern in (r'MEG 24..', r'EEG 05.'):
+    picks = mne.pick_channels_regexp(raw.ch_names, regexp=pattern)
+    raw.plot(order=picks, n_channels=len(picks))
```

Comment:
This is awesome ninja weaponry. But I am affraid this may be intimidating to coding novices. Can we just get the channels from the info['bads'] list?

Reply:
I think it's a useful feature that doesn't get introduced anywhere else, so I opted to give an expanded explanation of the regex rather than switch it to something simpler.